doc/manuals/user/walkthrough_apiv1.md

```diff
@@ -317,13 +317,13 @@ EOF
 
 The updateContext request payload contains a list of contextElement
 elements. Each contextElement is associated to an entity (whose
-identification is provided in the entityId element, in this case we
-provide the identification for Room1) and contains a list of
-contextAttribute elements ('attributes' for short, in JSON). Each
-contextAttribute provides the value for a given attribute (identified by
-name and type) of the entity. Apart from the list of contextElement
-elements, the payload includes also an updateAction element. We use
-APPEND, which means that we wa      nt to add new information.
+identification is provided by the id, type and isPattern fields, in this case
+we provide the identification for Room1) and contains  an attribute vector 
+with a list of attributes. Each element in the attributes list provides 
+the value for a given attribute (identified by name) of the entity. 
```

Comment:
I believe correct English would be "attribute list", not "attributes list". Not 100% sure, but almost :-)